### PR TITLE
Use @codait/max-facial-age-estimator

### PR DESCRIPTION
**Pull request by djalova**
Updates repo to use the model published on npm. Still need to do a few tests.
Addresses #9

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -10,12 +10,13 @@
...
 
-		<script src="https://cdn.jsdelivr.net/npm/@tensorflow/tfjs@0.15.3/dist/tf.min.js"></script>
+		<script src="https://cdn.jsdelivr.net/npm/@tensorflow/tfjs@1.2.2/dist/tf.min.js"></script>
+		<script src="https://cdn.jsdelivr.net/npm/@codait/max-facial-age-estimator"></script>
 		<script src="public/js/lib/stats.min.js"></script>
 		<script src="public/js/lib/cam.js"></script>
 		<script src="public/js/lib/pico.min.js"></script>
 		<script src="public/js/lib/face-api.min.js"></script>
-		<script src="public/js/ageEstimator.js"></script>
+		<!-- <script src="public/js/ageEstimator.js"></script> -->
```
Comment (vabarbosa) [inline index.html]: should probably just delete/remove instead of comment out

File: public/js/controller.js [hunk 2]
```diff
@@ -73,21 +72,55 @@ function rgba_to_grayscale(rgba, nrows, ncols) {
...
+
 /**
  * Takes in an array of bounding boxes and ages and updates the frame with the results.
  * @param {Array[Array[Number]]} faces          Array containing an array of numbers in the format [x, y, width, height]
  * @param {Array[Number]} ageResult             Array containing ages
  */
 function processAgeResults(faces, ageResult) {
-    for (i = 0; i < faces.length; ++i) {
+    for (i = 0; (i < faces.length && ageResult); ++i) {
+        console.log(ageResult);
```
Comment (vabarbosa) [inline public/js/controller.js]: maybe comment out or remove the `console.log` before merging?

**Review by vabarbosa — COMMENTED** [review]
looks good, just a couple minor comments

File: public/js/controller.js (hunk 2 above)
Comment (djalova) [reply public/js/controller.js]: Sure, I left it in because I was still testing a few things.

**Review by vabarbosa — APPROVED** [review]
LGTM

**Pull request merged**